style.css

```diff
@@ -0,0 +1,3 @@
+/* .form {
+  text-align: center;
+} */
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.

I trust you can remove this before merging.

Reply:
Thank You!

Yes, I will do that now, before merging.